src/check_ast.rs

```diff
@@ -533,6 +533,64 @@ where
                             },
                         )
                     }
+
```

Comment:
Forgot to update `StmtKind::Import`.

Reply:
Oops sorry missed this! A follow-up PR is fine.